src/branding-bar/BrandingBar.js

```diff
@@ -1,8 +1,62 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-const BrandingBarContainer = styled.header``;
+import LogoIconSrc from "../assets/icons/recidiviz_logo.svg";
+import { LinkPill } from "../pill";
+
+const brandingBarFlexProperties = css`
+  align-items: center;
+  display: flex;
+`;
+
+const BrandingBarContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 75px;
```

Comment:
> Re-adding this comment since it got blown away when I pushed my latest changes

Honestly, I just hard-coded this margin value to eyeball match the mocks. What would you think about adding a spacing section to the theme to allow us to use consistent spacing throughout the app? Something like what [Tailwind CSS](https://github.com/tailwindcss/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L132-L152) does. TBD what the actual values would be.

Reply:
That seems reasonable to me. Would like @macfarlandian to weigh in since I can't think of what the downside would be.